_posts/2019-06-27-quarkus-0-18-released.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: 'Quarkus 0.18.0 released'
+date: 2019-06-27
+tags: quarkus release
+author: gsmet
+---
+
+Quarkus 0.18.0 was just released and is now available on Maven Central.
+
+We usually release Quarkus every two weeks and it has just been a week since our last release so why release 0.18.0 so soon?
+Well, we have some good news to share: Quarkus now supports GraalVM 19 (requires it, even) and we wanted to put this version in your hands as quickly as possible.
+
+== What's new?
+
+Here are the main highlights of this release.
+
+=== GraalVM 19 Support
+
+When Oracle released GraalVM 19, it came with some breaking changes over the last RC.
+They also made some fundamental changes to the default behavior of GraalVM, for instance by not initializing all the classes by default at compile time.
+
+We had some adjustments to make on Quarkus but we also discovered some blocking issues in GraalVM itself.
+The Quarkus team worked together with the GraalVM team to get those fixed.
+
+With the release of GraalVM 19.0.2, you can now use Quarkus with GraalVM 19. Well, you have to, actually.
+
+Two important points:
+
+ * You need GraalVM 19.0.2+
+ * The `native-image` executable is not bundled in the GraalVM distribution anymore. You have to install it manually using `$GRAALVM_HOME/bin/gu install native-image`.
```

Comment:
s/You have to i/I/

Reply:
Ah ah, I heard your voice in my head, changed it and then put it back as I thought it was better to keep it.

Will change.